README.md

```diff
@@ -139,12 +139,12 @@ $ cd <your-repo-name>
 Finally, run the following command that will pull a pre-built image from DockerHub and will run your website.
 
 ```bash
-$ ./bin/dockerhub_run.sh
+$ docker-compose up
```

Comment:
does this mean the user has to install `docker-compose`? if yes, let's add "install docker compose" step to the list on lines 131-132 above.

Reply:
Yes. I also added `docker-compose` installation step.